Sources/SwiftKuery/Migration.swift

```diff
@@ -69,18 +81,38 @@ public class Migration {
         table1 = from
         table2 = to
     }
-    
-    /// Alter the name of the table to the name in the new version of the table.
-    ///
+    // MARK: Table Alteration
+
+    /**
+     Builds an executes an SQL query to alter the name of the old table to the name in the new version of the table.
```

Comment:
"Builds and" not "Builds an"

Reply:
This still needs fixing to ... "Builds and executes an SQL query..." not "Builds an executes an SQL query..."